fiftyone/operators/builtins/panels/model_evaluation/__init__.py

```
@@ -291,21 +310,36 @@ def load_evaluation(self, ctx):
         )
         if evaluation_data is None:
             info = ctx.dataset.get_evaluation_info(computed_eval_key)
+            evaluation_type = info.config.type
+            if evaluation_type not in SUPPORTED_EVALUATION_TYPES:
+                ctx.panel.set_data(
+                    f"evaluation_{computed_eval_key}_error",
+                    {"error": "unsupported", "info": serialized_info},
```

Ruff (0.8.2)

Undefined name `serialized_info`

(F821)